src/service/viewport-impl.js

```diff
@@ -794,42 +774,28 @@ export class ViewportBindingNaturalIosEmbed_ {
     // found is to reset scrolling on the AMP doc, which overrides natural BODY
     // scrolling with overflow:auto. We need the following styling:
     // html {
-    //   overflow: auto;
     //   -webkit-overflow-scrolling: touch;
     // }
     // body {
-    //   position: absolute;
-    //   overflow: auto;
-    //   -webkit-overflow-scrolling: touch;
+    //   overflow-x: hidden;
     // }
     setStyles(documentElement, {
-      overflowY: 'auto',
       webkitOverflowScrolling: 'touch',
     });
     setStyles(documentBody, {
       overflowX: 'hidden',
-      overflowY: 'auto',
-      webkitOverflowScrolling: 'touch',
-      position: 'absolute',
-      top: 0,
-      left: 0,
-      right: 0,
-      bottom: 0,
     });
 
     // Insert scrollPos element into DOM. See {@link onScrolled_} for why
     // this is needed.
     this.scrollPosEl_ = this.win.document.createElement('div');
     this.scrollPosEl_.id = '-amp-scrollpos';
     setStyles(this.scrollPosEl_, {
-      position: 'absolute',
```

Comment:
Removing the `position: absolute` on the `body` forced this. Because it's no longer positioned, any child that _is_ now positions itself relative to the viewport. So if we have a `position: absolute; top: 0; left: 0`, it will actually always be in the top-left corner of the screen.

> such as that padding will always be zero

Should we enforce that?

> any kind of reordering would be done

That's the same requirement as `endPosEl_`, so I didn't think it'd be an issue.